java/code/src/com/redhat/rhn/domain/ContentFilterEntity.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2024 SUSE LLC
```

Comment:
I this should be 2025, I guess :slightly_smiling_face: . Check your licence settings.